Sources/XCHammer/XcodeBuildSystemInstaller.swift

```diff
@@ -35,7 +35,7 @@ enum XcodeBuildSystemInstaller {
         // Check each Xcode that's compatible with this version and verify if
         // it's installed
         var hasUnlinkedXcodes = false
-        let cmd = xcodeLocatorPath + " 2>&1 | grep expanded=11 | sed -e 's,.*file://,,g' -e 's,/:.*,,g'"
+        let cmd = xcodeLocatorPath + " 2>&1 | grep expanded=13 | sed -e 's,.*file://,,g' -e 's,/:.*,,g'"
```

Comment:
I wonder why it needs this line - theoretically the installer should run for all versions

Reply:
Maybe because the Xcode locator runs in a different step in a bash script and not in swift? 

We could find a way to pass as input the list of found Xcodes too

Reply:
Actually nvm it's running the locator, maybe we can do the same we're doing in xcbuildkit here and loop through the versions 

Will take a look

Reply:
@jerrymarino just added/tested the logic to install for supported Xcodes (same logic we run in `xcbuildkit`), see this commit: https://github.com/bazel-ios/xchammer/pull/31/commits/e1a263e7af9ae1f666c35ba8795d72256885fbd8

Reply:
landing for now, if there's anything else to address here I can put up a follow up